internal/test/wmcb/wmcb_test.go

```diff
@@ -143,22 +196,23 @@ func (vm *wmcbVM) initializeTestBootstrapperFiles() error {
 }
 
 // remoteDownloadExtract downloads the tar file in url to the remoteDownloadFile location and checks if the SHA matches
```

Comment:
Comment doesnt match function name

Reply:
I'd argue that this function should be called something like `checksumRemoteFile`, since the end goal isnt to download it.

Reply:
This function is downloading the tar file and doing the checksum. So calling it `checksumRemoteFile` is not sufficient just because the checksum check is the last thing it does. In fact if a SHA is not provided then it just downloads the file. I am open to other names.